Makefile

```diff
@@ -0,0 +1,23 @@
+run:
+	cargo run $(path)
```

Comment:
Usually, when we want to run a program, we use `cargo run --release` to optimize the performance. `cargo run` should be only used when developing